metacat-main/src/main/java/com/netflix/metacat/main/services/DatabaseService.java

```diff
@@ -24,7 +24,8 @@
      * Gets the database for the given name.
      * @param name name
      * @param includeUserMetadata if true, will also include the user metadata
+     * @param includeTableNames   if true, include the list of table names
      * @return database info
      */
-    DatabaseDto get(QualifiedName name, boolean includeUserMetadata);
+    DatabaseDto get(QualifiedName name, boolean includeUserMetadata, boolean includeTableNames);
```

Comment:
Since this seems to be true most places would it be cleaner just to make a separate interface `getWithoutTableNames` to make it very clear? The inside the implementation you can have a private method with the boolean switch

Reply:
This looks much simpler and inline with 'includeUserMetadata' flag than adding another method.